cost/azure/reserved_instances/utilization_mca/azure_reserved_instance_utilization_mca.pt

```diff
@@ -0,0 +1,205 @@
+      field "utilizationAggregates", jq(col_item, ".properties.utilization.aggregates")
+      field "quantity", jq(col_item, ".properties.quantity")
+      field "expiryDate", jq(col_item, ".properties.expiryDateTime")
+      field "appliedScopeType", jq(col_item, ".properties.appliedScopeType")
+    end
+  end
+end
+
+datasource "ds_reservations_below_utilization" do
+  run_script $js_get_reservations_below_utilization, $ds_revervation_all, $param_lookback_period, $param_utilization
+end
+
+script "js_get_reservations_below_utilization", type: "javascript" do
+  parameters "ds_reservation_all", "param_lookback_period", "param_utilization"
+  result "result"
+  code <<-EOS
+    var result = [];
+
+    var grain = 30;
+    
+    if (param_lookback_period == "Last 7 days") {
+        grain = 7;
+    }
+    
+    _.each(ds_reservation_all, function (reservation) {
+        var utilizationValue = _.findWhere(reservation.utilizationAggregates, { grain: grain }).value;
+    
+        if (utilizationValue < param_utilization) {
+            reservation.reservationOrderId = reservation.id.replace("/providers/microsoft.capacity/reservationOrders/", "").replace("/reservations/" + reservation.reservationId, "");
+            reservation.lookBackPeriod = grain + " days";
+            reservation.utilizationPercentage = utilizationValue;
+    
+            result.push(reservation);
+        }
+    });
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_azure_active_reserved_instances" do
+  validate $ds_reservations_below_utilization do
+    summary_template "{{ len data }} Azure Reserved Instances below {{ parameters.param_utilization }}% of utilization - {{ parameters.param_lookback_period }}"
+    escalate $esc_email
+    check false
+    export do
+      field "displayName" do
+        label "Display name"
+      end
+      field "appliedScopeType" do
+        label "Applied scope type"
+      end
+      field "expiryDate" do
+        label "Expiry date"
+      end
+      field "quantity" do
+        label "Quantity"
+      end
+      field "reservedResourceType" do
+        label "Reserved resource type"
+      end
+      field "reservationId" do
+        label "Reservation ID"
+      end
+      field "reservationOrderId" do
+        label "Reservation order ID"
+      end
+      field "location" do
+        label "Region"
+      end
+      field "sku" do
+        label "Instance type"
+      end
+      field "term" do
+        label "Term"
+      end
+      field "instanceFlexibility" do
+        label "Instance flexibility"
+      end
+      field "billingScopeId" do
+        label "Purchasing subscription"
+      end
+      field "billingPlan" do
+        label "Reservation billing plan"
+      end
+      field "lookBackPeriod" do
+        label "Look Back Period"
```

Comment:
Please, rename field to "lookbackPeriod" and label to "Lookback Period"so it goes in line with normalization in the future.